### ISSUE TITLE
Issue: Lunar Deathworm and Calicia do not get recognized

### ISSUE THREAD

**Issue by Styx-Blackwell** [InitialDet/AutoHook#77]
Hi! I have found that for me Calicia and Lunar Deathworm are no longer recognized by the plugin. Considering I need the names of the fish in order to make Hook Presets this unfortunately presents an issue. 
I have found a way around the problem for now however I thought letting you know would be good.
![image](https://user-images.githubusercontent.com/95082748/222770241-d07436aa-2be8-4b77-8e7f-d16c20248fa9.png)
![image](https://user-images.githubusercontent.com/95082748/222770434-576997bd-a1f3-484a-b44e-5cd651c9e445.png)

**Comment by InitialDet** [comment]
The name of the fish appears to be lowercase in the chat, like "calicia" and not "Calicia". That might be the case.

Also, what was the way around you found

**Comment by Styx-Blackwell** [comment]
![image](https://user-images.githubusercontent.com/95082748/223101515-9f885a1b-2a83-41c1-983d-2a4c4e02c492.png)
Hmm, I caught a calicia again last but the plugin doesn't seem to recognize it. Maybe cause I'm using Chat 2? I mean cause it's not showing at "Current bait/fish:"

**Comment by InitialDet** [comment]
![image](https://user-images.githubusercontent.com/13919114/223281317-4c9aa1f4-f8bd-4f20-b170-344f1af59aac.png)

I'm doing some tests and it appears to be working normally here
type /xllog in the chat and you should see a screen similar from mine, it prints in the log the current config being used when Cast is used.

Try removing Chat2 and restarting your game, that might be the problem?

**Comment by Nheat** [comment]
I've also encountered this bug, but with any fish, and not all the time. Custom Presets just no longer recognizing any fish and only showing "-" for current mooches and last catches. Not sure what causes it. Log just shows it using default config, and I don't have Chat2.

**Comment by InitialDet** [comment]
I tried to reproduce this but no success at the moment, i'm not sure if this might be a conflict with another plugin or it happens only when doing specific steps

**Comment by InitialDet** [comment]
I checked the code and found some possible solutions for this, im unable to test it myself so please tell me if it works (v.2.4.4.1)

**Comment by Nheat** [comment]
Got the bug again. If it helps, I'll list what I did after TP'ing to the area: Mounted, flew towards fishing hole (North Drybone), retrieved bait (Syrphid Basket) from Saddlebag and configured AutoHook on the way, landed, applied bait (no bait applied prior), manually Casted into the hole (at which point I realized I had not yet discovered the hole), caught and Surface Slapped the first thing that bit, noticed from here that AutoHook was only showing "-" for my last catch, enabled AutoHook (nothing happened in this case, sometimes it starts on its own as soon as I enable, sometimes only after I Cast), manually Chum'd, manually Casted, and let it do its thing from there. At no point did it ever record my last catch.

I thought it might be related to it being an undiscovered hole, so I tried putting away my rod and recasting at the same hole, then flying to a different hole in the same area, but it still wasn't working.
